templates/Deepcell_Postprocessing.ipynb

```diff
@@ -0,0 +1,300 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Both channels and Point8\n",
+    "\n",
+    "# Get input for overlay\n",
+    "input_data_xr = load_utils.load_imgs_from_multitiff(deepcell_input_dir,\n",
+    "                                                    multitiff_files=fovs_input)\n",
+    "# Overlaying the DNA\n",
+    "overlay_channels = input_data_xr.channels.values\n",
+    "warnings.simplefilter(\"ignore\")\n",
+    "\n",
+    "# Channel 0 and Point 8\n",
+    "segmentation_utils.visualize_segmentation(\n",
+    "            segmentation_labels_xr=segmentation_labels,\n",
+    "            fovs=input_data_xr[:,:,:,0].fovs.values, channel_data_xr=input_data_xr,\n",
+    "            chan_list = overlay_channels[0], output_dir=viz_dir, show=True)\n",
+    "\n",
+    "chan_list = overlay_channels[0]"
+   ]
+  },
```

Comment:
@ackagel I'm forgetting, did we also want a run where we visualize multiple fovs?

Reply:
According to Noah's earlier comment, not within the notebook.